impress-org/givewp
# Campaigns Featured Image block now syncs with the post's featured image

Resolves #5283

## Description



- Wraps `PostFeaturedImage` from `@wordpress.editor`.
- Overrides the placeholder size to fit the layout.

## Visuals



![Peek 2020-09-22 14-05](https://user-images.githubusercontent.com/10858303/93920154-c6690f00-fcdc-11ea-952e-452d750a07e7.gif)


![Screenshot_2020-09-22 Edit Post ‹ WordPress — WordPress](https://user-images.githubusercontent.com/10858303/93919519-d03e4280-fcdb-11ea-99ec-f8716d7244e7.png)

![Screenshot_2020-09-22 Edit Post ‹ WordPress — WordPress(1)](https://user-images.githubusercontent.com/10858303/93919511-cf0d1580-fcdb-11ea-98e2-3bfff7fdd8db.png)

## Pre-review Checklist




## Testing Instructions



- Create a Campaign
- Click on the Featured Image block to update the featured image.
- - Featured Image in sidebar should be updated.
- Click on the Featured Image in the sidebar to update the featured image.
- - Featured Image block should be updated.
- Removing the Featured Image from the sidebar removes the featured image form the block.
